docs/writing-stories/index.mdx

```diff
@@ -204,13 +204,13 @@ Addons can enhance args. For instance, [Actions](../essentials/actions.mdx) auto
 
 ### Using the play function
 
-Storybook's `play` function and the [`@storybook/addon-interactions`](https://storybook.js.org/addons/@storybook/addon-interactions) are convenient helper methods to test component scenarios that otherwise require user intervention. They're small code snippets that execute once your story renders. For example, suppose you wanted to validate a form component, you could write the following story using the `play` function to check how the component responds when filling in the inputs with information:
+Storybook's `play` function and the [Component tests panel](../writing-tests/component-testing.mdx) are convenient helper methods to test component scenarios that otherwise require user intervention. They're small code snippets that execute once your story renders. For example, suppose you wanted to validate a form component, you could write the following story using the `play` function to check how the component responds when filling in the inputs with information:
 
 {/* prettier-ignore-start */}
 
 <CodeSnippets path="login-form-with-play-function.md" />
 
-Without the help of the `play` function and the `@storybook/addon-interactions`, you had to write your own stories and manually interact with the component to test out each use case scenario possible.
+Without the help of the `play` function and component tests, you had to write your own stories and manually interact with the component to test out each use case scenario possible.
```

Comment:
As you're removing the references to `@storybook/addon-interactions` from the documentation, including the links to the addon catalog, it would be helpful to remove them from the other docs sources in the monorepo (i.e., README files) such as [this](https://github.com/storybookjs/storybook/tree/next/code/core/src/test) one. Wouldn't you agree?